atd-vzd/schema/engineering_areas.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE public.engineering_areas(
+    id integer NOT NULL,
+    eng_area_label text,
+    eng_area_geometry geometry,
```

Comment:
What do you think about calling these columns something like "label" and "geometry" instead of repeating the table name in an abbreviated form? Since the table only holds engineering areas, I think you could get away with less specific column names.

Reply:
yeah i can definitely make these less specific, i never know whats a better route for column names but that makes sense to me!

Reply:
Awesome - I think our queries will be more readable in the future because we'll be doing stuff like `engineering_areas.label` which captures what's in the column really well. Thanks!